protocols/grpc/utils.go

```diff
@@ -287,3 +299,86 @@ func ResourcesGRPCtoOCI(grpcResources *LinuxResources) (*specs.LinuxResources, e
 
 	return s, err
 }
+
+// ChangeToBundlePath changes the cwd to the bundle path defined in the OCI spec
+func ChangeToBundlePath(spec *specs.Spec) (string, error) {
```

Comment:
These utils functions might be better to move out from grpc package.

You know, this package can be vendored later to kata-runtime, because kata-runtime will be client and communicate with kata-agent via GRPC protocol, they share same GRPC protocol definitions.

So if these functions do no help to runtime, which means it can't help kata-runtime do GRPC client operations, it might be better to move out from this package.